test/api_image_save_and_load_test.go

```diff
@@ -25,20 +25,20 @@ func init() {
 func (suite *APIImageSaveLoadSuite) SetUpTest(c *check.C) {
 	SkipIfFalse(c, environment.IsLinux)
 
-	PullImage(c, helloworldImage)
+	PullImage(c, busyboxImage125)
```

Comment:
can we use image `busyboxImage128`?

Reply:
sure. will do

Reply:
done. I will file other pr to replace all the busyboxImage125 because the 128 tag is stable.

Reply:
I roll it back because 125 tag has only one manifest, not a list.

Reply:
I think define `busyboxImage` is enough, without tag, or we need change lots of places, we have done these times

Reply:
we cannot use the latest one, because the latest has no stable digest and ID. we don't know what the real ID is.

Reply:
What I mean is `busyboxImage=xxx125 or xxx128`, I do not like `busyboxImage128` define with tag at all. It can be fix in another pr.

Reply:
Or can we reset these change in test.

Reply:
I will change it to `busyboxLegacyImage` instead of `125`.